config/uitests-amo.js

```diff
@@ -0,0 +1,19 @@
+// For use on addons-server, or with the command `amo:ui-tests` and the
+// HOSTNAME  variable as `ui-tests`.
```

Comment:
I don't think we should mention the `HOSTNAME` variable (in part because we load the config _via_ `NODE_CONFIG_ENV`)